cmd/gokeyless/initialize.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"strings"
+
+	"github.com/cloudflare/cfssl/csr"
+	"github.com/cloudflare/cfssl/log"
+	"github.com/cloudflare/gokeyless/server"
+)
+
+type v4apiError struct {
+	Code    json.Number `json:"code,omitempty"`
+	Message string      `json:"message,omitempty"`
+}
+
+type initAPIResponse struct {
+	Success  bool              `json:"success,omitempty"`
+	Messages []string          `json:"messages,omitempty"`
+	Errors   []v4apiError      `json:"errors,omitempty"`
+	Result   map[string]string `json:"result,omitempty"`
+}
+
+var apiEndpoint = "https://apix.cloudflare.com/certificate"
```

Comment:
This should maybe be a parameter. It will be `https://apix.cloudflare.com/certificate` soon but currently it's `https://apix.svc.cfdata.org/certificate`...